scripts/controller/ROS_controller.py

```diff
@@ -186,11 +188,13 @@ def wrapper(self, *args,**kwargs):
             elif self.auth == self.node_name:
                 ret = func(self, *args,**kwargs)
             else:
-                ret = ""
                 #rospy.logwarn("This node don't have authority...")
                 self.log.warn("This node don't have authority...")
                 print("current authority : ", self.auth)
-                pass
+                raise NECSTAuthorityError(
```

Comment:
`NECSTAuthorityError`は本当にauthority errorが出ている場合と、タイムラグが原因で出る場合があります。後者の場合は少し待ってからもう一回実行すると治り?ます。エラーメッセージに注意を促す記述をする、もしくは `self.auth`で返される文字列に応じてerrorの中身を変更してください。

Reply:
The message is now improved;
- when `self.auth` is empty, instruction `This might be caused by software/network lag. Wait a moment and retry.` is printed
- otherwise name of currently controlling node is printed